db_proposal.md

```diff
@@ -0,0 +1,34 @@
+Database Layout Proposal
+========================
+This is my proposal for the db layout.  Thoughts?
+
+Company
+========
+* Ticker (string, unique)
+* Name (string)
+* Industry (string)
+* Sector (string)
+
+PricePoint (historical, current)
+===========
+* Company::Ticker (foregin key, has_one)
+* Price (number)
+* Datetime (datetime, utc?)
+
+User
+====
+* name
+* email
+* password
+* favorited stocks (foreighn key to Company::Ticker, has_many)
+
+Tweet (current)
+=====
+* content (string)
+* author (string) [MAYBE?]
```

Comment:
Yes on author. It looks nicer for UI and ethos of the tweets. Since we're querying for recent and popular right now, it might be nice to put the timestamp with the tweet as well. Or perhaps show the popular ones first...